src/main/java/org/launchcode/mentalhealthcareaccess/models/data/dto/RegisterFormDTO.java

```diff
@@ -0,0 +1,64 @@
+package org.launchcode.mentalhealthcareaccess.models.data.dto;
+
+import javax.validation.constraints.NotBlank;
+
+public class RegisterFormDTO extends LoginFormDTO {
+    @NotBlank
+    private String verifyPassword;
+    @NotBlank
```

Comment:
This annotation will prevent a company (w/o first or last name) from registering, since leaving the first name field blank will result in a validation error. I suggest removing this, and then doing some custom validation within the post handler. There, you can check that *either* company name is not blank *or* both first and last names are not blank.